src/graph-window.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<script src="lib/VSS.SDK.js"></script>
+		<script src="work-item-provider.js"></script> 
+		<script src="renderer.js"></script>
+		<script src="work-item-graph.js"></script>
+		<script src="vis/vis.js"></script>	
+		<link href="vis/vis-network.min.css" rel="stylesheet" type="text/css"></link>
+		<style type="text/css">
+            #refreshButton {
+                position: absolute;
+                left: 20px;
+                top: 20px;
+                z-index: 1;
+				border: 2px solid rgba(43, 124, 233, 1);
+                background-color: rgba(229, 233, 234, 1);
+            }
+			#network {
+				height: 100vh;
+				width: 100vw;
+				border: 1px solid lightgray;
+				background-color: rgba(153, 182, 229, 0.4);
+			}
+		</style>
+	</head>
+	<body>
+		<div id="network"></div>
+		<script>
+			var parentWorkItemId;
```

Comment:
We've generally preferred to put all script code in .JS files instead of in HTML files. Makes it easier to do linting, write tests, keep it formatted nicely, etc. Also gives you more flexibility in controlling when the script is downloaded and executed at page startup.